BDArmory/Modules/BDAdjustableArmor.cs

```diff
@@ -325,6 +325,7 @@ void UpdateThickness()
             if (armor != null && armorTransforms != null)
             {
                 armorthickness = Mathf.Clamp((armor.Armor / 10), 0.1f, 1500);
+                if (isTriangularPanel && part.isMirrored) armorthickness = Mathf.Abs(armorthickness) * -1;
                 //if (!isCurvedPanel)
```

Comment:
needs further work, but that will take more time if we want to get an update out in the near future